__tests__/app.test.js

```diff
@@ -18,7 +18,7 @@ describe("/", ()=>{
 			.get("/")
 			.expect(200)
 			.then(({body}) =>{
-				expect(Object.keys(body.endpoints)).toEqual([
+				expect(Object.keys(body.endpoints)).toStrictEqual([
```

Comment:
I probably wouldnt test this that much, i'd just check you get an object. 
At the moment, this is a brittle test and will break everytime you update the file.